workflow-casava/workflow/config/workflow.ini

```diff
@@ -14,4 +14,6 @@ threads=1
 memory=4000
 do_olb=0
 
+provision_out_undetermined=true
```

Comment:
This is the right way around. Can you make the decider reflect this please.

Reply:
I'll update the decider to use "--provision-out-undetermined=false"